_posts/en/2020-12-10-micro-acquisition-pilot-update1.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title:  "Micro-Acquisition Pilot status update 1"
+ref: micro-acquisition-update1
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team and Micro-Acquisition Pilot team members"
+date: "2020-12-10"
+---
+
+This is our first status update for the Micro-Acquisition Pilot that we are working on at ESDC. We will continue to provide updates as the project progresses.
+
+## Approvals and Governance
+
+- The pilot is being co-sponsored by the Innovation and Information Technology Branch (IITB) and Chief Financial Officer Branch (CFOB) at ESDC. The CFOB team have briefed up to the CFO.
+
+- Both ESDC CFOB and IITB have committed FTE resources to the one year pilot as part of the project documents currently going for approval
+
+## Process
+
+- ESDC CFOB has confirmed that acquisition cards/PayPal can be used for payment of suppliers. While the details on that are still to be worked out, paying by credit card will help achieve 2 of our expected results for the pilot:  
+
+  - Reduced burden on procurement officers (& simplifying the procurement process): credit card purchases can be done without CFOB's intervention as only the person posting the micro-acquisition opportunity and the person holding the credit card will be involved). CFOB will be helping to establish guardrails.
+
+  - Faster payment to suppliers
+
+## Timeline
+
+- An official go live date is targeted for May 2021 to ensure adequate time to design and test the process, website, and communications
+
+- We are meeting with potential clients (within ESDC) to assess their requirements and help them plan their budget for next Fiscal Year
+
+## Possible Challenges
+
+- we may experience delays due to backlogs in the project approval process
+
+## Collaborative efforts
+
+- We are working with the [BC Dev Exchange](https://bcdevexchange.org/) team to help us develop a 'bid evaluation process' which is both fair to suppliers and not onerous on ESDC clients.  They have cautioned us against going priced-based and this matches the research we have done re: the US' experience in micro-purchasing (leads to $1 bids from larger companies to eliminate smaller companies from winning).  
+
+We are also looking to follow the lead of the BC Dev Exchange and establish anonymous bid evaluation.
```

Comment:
I think there would need to be bullet points here? Could you please confirm @RachelMuston? Thanks! :+1:

Reply:
Good q.  They aren't really separate bullets as they are all part of the same consultation with the BC Dev Exchange.  I separated them as it was such a big block of text.  But maybe it would be better organized with headers?  E.g. H3 for BC Dev Exchange, SSC and OSME. and then bullets under each H3.  BC Dev Exchange would have 3 bullets, the others would have one bullet.